src/components/calcite-handle/calcite-handle.scss

```diff
@@ -1,30 +1,29 @@
 :host {
-  display: flex;
+  @apply flex;
 }
 
 .handle {
-  display: flex;
-  align-items: center;
-  align-self: stretch;
-  justify-content: center;
-  padding: var(--calcite-spacing-three-quarters) var(--calcite-spacing-quarter);
-  background-color: var(--calcite-ui-foreground-3);
-  background-color: transparent;
-  border: none;
-  color: var(--calcite-ui-border-3);
+  @apply flex 
+    items-center 
+    self-stretch 
+    content-center 
+    bg-transparent 
+    text-color-2 
+    bg-foreground-3 
+    bg-transparent 
+    border-none 
+    cursor-move;
+  padding: theme("spacing.3") theme("spacing.1");
+  @apply cursor-move;
   line-height: 0;
-  cursor: move;
   &:hover {
-    background-color: var(--calcite-ui-foreground-2);
-    color: var(--calcite-app-foreground);
+    @apply bg-foreground-2 text-color-2;
   }
-  @include focus-style-base();
+  @apply focus-base;
```

Comment:
move `focus-base` into the existing @apply rule above

Reply:
good catch, done